site/src/atomics/spacing.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+Spacing Atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+## Class Pattern
+
+`<css-property-name>-<value>-<screensize>`
+
+For general information about the pattern, visit the [Atomics page](https://github.com/microsoft/atlas-design/blob/main/css/src/atomics/README.md).
+
+The following `<css-property-names>-<values>-<screensizes>` are supported.
+
+Accepted property names
+
+- `margin`, `padding`
+- `margin|padding-inline`
+- `margin|padding-block`
+- `margin|padding-top|right|bottom|left`
+
+Accepted values
+
+- `none, xs, s, m, l, xl, xxl, xxxl`
+
+Accepted screensizes
+
+- `tablet, desktop, widescreen`
+
+## Usages
+
+Adding uniform spacing
+
+Use `margin` with a size value to add spacing around an element.
+
+```html
+<div class="margin-xs border">XS</div>
+<div class="margin-s border">X</div>
+<div class="margin-m border">M</div>
+<div class="margin-l border">L</div>
+<div class="margin-xl border">XL</div>
+<div class="margin-xxl border">XXL</div>
+<div class="margin-xxxl border">XXXL</div>
+```
+
+Adding horizontal and vertical spacing
+
+Use `margin-inline` with a size value to apply both left and right spacing. Use `margin-block` for top and bottom spacing.
+
+```html
+<div class="margin-inline-xl border">Margin-inline</div>
+<div class="padding-inline-xl border">Padding-inline</div>
+<div class="margin-block-xxl border">Margin-block</div>
+<div class="padding-block-xxl border">Padding-block</div>
+```
+
+Adding spacing to a side
+
+Use `margin-left` with a size value to add left margin to an element.
```

Comment:
To match what is actually shown in the example, let's update the sentence to include the `right` side as well:
Use `margin-left` \ `margin-right`  with a size value to add left\right margin to an element.

Reply:
Updated